code/testapp/src/main/java/com/adobe/analyticstestapp/AnalyticsTestApp.java

```diff
@@ -56,6 +57,7 @@ public void fail(AdobeError adobeError) {
             }
         });
 
+
```

Comment:
maybe remove the old registration and 1.x dependencies, we can add them locally for testing